content/docs/07-reference/ai-sdk-ui/03-use-object.mdx

```diff
@@ -52,15 +52,15 @@ export default function Page() {
     },
     {
       name: 'schema',
-      type: 'ZodSchema<RESULT>',
+      type: 'Zod Schema | JSON Schema',
```

Comment:
is this supposed to have a space?

Reply:
yes I didn't want to make it a real type, otherwise it would look like `z.Schema<T> | Schema<T>` which can be confusing

Reply:
ack! all good then - just wanted to make sure it wasn't a typo